src/main/java/com/aws/greengrass/lambdatransformer/common/models/LifecycleType.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.lambdatransformer.common.models;
+
+public enum LifecycleType {
+    Install,
+    Run,
+    Standby,
+    Restart,
+    startup,
+    shutdown,
+    setenv
```

Comment:
nit: inconsistent casing...

Reply:
this is how it is in the `EvergreenComponentManagementService` so that's how I'm keeping it